sdk/storage/azure-storage-blob/azure/storage/blob/_generated/models/_azure_blob_storage_enums.py

```diff
@@ -167,6 +167,7 @@ class StorageErrorCode(str, Enum):
     lease_not_present_with_container_operation = "LeaseNotPresentWithContainerOperation"
     lease_not_present_with_lease_operation = "LeaseNotPresentWithLeaseOperation"
     max_blob_size_condition_not_met = "MaxBlobSizeConditionNotMet"
+    no_authentication_information = "NoAuthenticationInformation"
```

Comment:
I think this code needs to be added to the Error code enum in the `_shared` directory:
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py#L19